src/signals/incident/components/form/MapSelectors/Asset/Summary/Summary.tsx

```diff
@@ -36,7 +36,7 @@ const Summary: FC = () => {
   const { description } =
     meta.featureTypes.find(({ typeValue }) => typeValue === type) ?? {}
 
-  const summaryDescription = `${description}${id ? ` - ${id}` : ''}`
+  const summaryDescription = [description, id].filter(Boolean).join(' - ')
   let summaryAddress = coordinates ? 'Locatie is gepind op de kaart' : ''
```

Comment:
This change does not make it more readable to me.

Reply:
What would you suggest?

Reply:
to keep it the way it was

Reply:
That's not possible, because in some cases, `description` is `undefined`. That means we need to filter it out. And that's exactly what this change does.